packages/cache-core/src/cacheHandler.ts

```diff
@@ -132,6 +132,10 @@ export class Cache implements CacheHandler {
         return null
       }
 
+      if (data.value?.kind === 'ROUTE') {
```

Comment:
just to confirm that this type is the same for app router and page router

Reply:
no, looks like for pages router we don't have ability to cache API calls. 
it doesn't trigger set-handler when we call /api endpoint